triton-vm/src/stark.rs

```diff
@@ -920,6 +929,17 @@ impl Stark {
                 summands.push(quotient_shifted);
             }
 
+            for arg in EvalArg::all_private_evaluation_arguments().iter() {
```

Comment:
@sshine, I have a feeling that the code duplication going on here can be avoided somehow by leveraging the fact that both `PermArg` and `EvalArg` implement the trait `CrossTableArg`. Ideally, I'd like to do something akin to:
```rust
let all_things_linking = [PermArg::all_args(), EvalArg::all_args()].concat();
for arg in all_things_linking.iter() {
    …
}
```
For reasons of type safety, this doesn't work. Is there a way that _does_ work?

Reply:
Resolved in 7b95c90